scripts/build.js

```diff
@@ -83,7 +79,8 @@ const runE2eTests = async () =>
         // NOTE: Cypress uses the exit code to show the number of failed tests.
         // If we do not exit here, the child process (that's running the test) will not be able to feedback
         // to Github Actions if the tests pass.
-        process.exit(exitCode)
+        // If exitCode is null, means that the child process was killed by a signal.
+        process.exit(exitCode || 1)
```

Comment:
Hmm are we unintentionally changing the semantics of the code here? 
the comment mentions that the exit code is used to show the number of failed tests, if the child process was killed by a signal, should we report that 0 tests passed rather than 1?

Reply:
Hmm we can't do an `exit 0` as it would mean the build passed, so I opted to default to `1` so that the build can fail correctly when it is killed externally.